README.md

```diff
@@ -111,8 +111,8 @@ fizz.WithoutSecurity()
 // Add a Code Sample to the operation.
 fizz.XCodeSample(codeSample *XCodeSample)
 
-// Mark the operation as internal or external.
-fizz.XInternal(isInternal bool)
+// Mark the operation as internal.
```

Comment:
Since this is an extension, it might be beneficial for the readers to indicate that this is interpreted by third-party tools, and the impact is only on visual rendering. No need for an essai :) just a quick note.

Reply:
Yes, that can be helpful. Added a quick note now.